.github/workflows/validate.yml

```diff
@@ -0,0 +1,21 @@
+name: validate
+on:
+  pull_request:
+    branches:
+      - "**"
+  push:
+    branches:
+      - "main"
+jobs:
+  check-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm install
+      - run: npm test
```

Comment:
I thought about that. However, since we also want to know if `npx tree-sitter generate` causes changes, it just seemed less fragile to run `npm test` instead of `bin/generate-corpus` and `bin/generate-parser`. But happy to make more specific.